README.md

```diff
@@ -0,0 +1,45 @@
+# Galoy CLI
+Galoy CLI is a Rust-based CLI client that can interact with the Galoy backend using GraphQL queries and mutations.
+
+## Installation
+To install Galoy CLI locally and set up a local environment:
+
+1. Clone the repository using git clone `https://github.com/GaloyMoney/galoy-cli.git`.
+2. Install Rust in your local machine and run cargo build to build all binary and library targets of the selected packages.
+3. Run cargo run command to run all tests of the Galoy CLI repository and see the usage, commands, and options available to interact.
+4. Interact with the CLI yourself to become familiar with it. After building, try the following command:
+
+`GALOY_API=https://api.staging.galoy.io/graphql ./target/debug/galoy-client getinfo`
```

Comment:
ideally we would want to highlight that the backend can also be run locally by running https://github.com/GaloyMoney/galoy (so you don't have to fetch our staging environment). the captcha can also be deactivated with the local dev environment

Reply:
So i need to remove this?

Reply:
@nicolasburtey i think i should update the readme with providing the link to galoy backend(https://github.com/GaloyMoney/galoy) in place of these commands.